File: Ejercicios Corredor/Trabajo Corredor/Booleano.js
class Booleano {
  activo = true;
  encendido = false;
  aprobado = true;
  disponible = false;
  visible = true;

  esVerdadero(valor) { return valor; }
  negacion(valor) { return !valor; }
  ambosVerdaderos(a, b) { return a && b; }
  algunoVerdadero(a, b) { return a || b; }
  iguales(a, b) { return a === b; }
}

File: Ejercicios Corredor/Trabajo Corredor/Booleano.java
public class Booleano {
    boolean activo = true;
    boolean encendido = false;
    boolean aprobado = true;
    boolean disponible = false;
    boolean visible = true;

    public boolean esVerdadero(boolean valor) { return valor; }
    public boolean negacion(boolean valor) { return !valor; }
    public boolean ambosVerdaderos(boolean a, boolean b) { return a && b; }
    public boolean algunoVerdadero(boolean a, boolean b) { return a || b; }
    public boolean iguales(boolean a, boolean b) { return a == b; }
}
